build-output-api/on-demand-isr/.vercel/output/functions/revalidate.func/index.js

```diff
@@ -0,0 +1,46 @@
+const https = require('https')
+
+// The bypass token should be a build-time randomly generated string of at least 32 characters.
+// This value should only be exposed in the function config and inside of the function itself.
+// DO NOT expose this value on the client-side or a malicious user could trigger boundless revalidations
+const bypassToken = '87734ad8259d67c3c11747d3e4e112d0'
+
+module.exports = (req, res) => {
+  const host = req.headers.host
+  const deployedUrl = `https://${host}`
+
```

Comment:
We need a guard here showing this request should be authenticated e.g.

```js
if (process.env.USER_SECRET !== req.headers.authorization) {
  res.statusCode = 403
  res.end('Not Authorized')
}
```

Reply:
I add a really basic check with a comment. How's that look?

Reply:
This seems alright if the goal is to allow users to actually trigger the revalidation on the demo without knowing a private secret/login